### PR TITLE
Issue 163: Include simple-mock as a static dependency for test-helper

### PR DESCRIPTION
**Pull request by OldSneerJaw**
With this change, the test-helper module no longer depends on any external libraries, making it easier to use as a dependency in other projects.

Addresses issue #163.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,4 +2,3 @@ language: node_js
 node_js:
   - "node"
   - "lts/*"
-  - "0.10"
```
Comment (OldSneerJaw) [inline .travis.yml]: Node.js v0.10.x is completely out of date and there is no need to be able to build the project with it anymore. In fact, this version was already removed on the `master` branch [a few months ago](https://github.com/Kashoo/synctos/commit/dafa61aab3c8202e68fc73a829d9fb451afd6b71#diff-354f30a63fb0907d4ad57269548329e3). This change should hopefully put an end to the intermittent spurious build failures on this branch.

File: lib/simple-mock/LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
Comment (OldSneerJaw) [inline lib/simple-mock/LICENSE]: Copied verbatim from https://github.com/jupiter/simple-mock/blob/v0.8.0/LICENSE.

File: lib/simple-mock/index.js [hunk 3]
```diff
@@ -0,0 +1,240 @@
+/* global define */
```
Comment (OldSneerJaw) [inline lib/simple-mock/index.js]: Copied verbatim from https://github.com/jupiter/simple-mock/blob/v0.8.0/index.js.

File: etc/sync-function-template.js [hunk 4]
```diff
@@ -82,7 +82,8 @@ function synctos(doc, oldDoc) {
       // is enabled and the document was deleted via the Couchbase SDK. Skip everything else and simply assign the
       // public channel
       // (https://developer.couchbase.com/documentation/mobile/current/guides/sync-gateway/channels/index.html#special-channels)
-      // to the document so that it can be replaced in the future.
+      // to the document so that users will get a 404 Not Found if they attempt to fetch (i.e. "view") the deleted
+      // document rather than a 403 Forbidden.
```
Comment (OldSneerJaw) [inline etc/sync-function-template.js]: Updated these comments to address [this](https://github.com/Kashoo/synctos/pull/162/files#r159466132).

Comment (dkichler) [reply etc/sync-function-template.js]: 👍

**Pull request merged**